mj_models/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Is there any specific reason we need `3.5` instead of maybe `3.2`? I

Reply:
It might be only my machine, but I get:
`CMake Deprecation Warning at mujoco_ros/mujoco_ros/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
`